### PR TITLE
Release 7.1.0

### PR DESCRIPTION
**Pull request by danirabbit**
Todo:
- [x] I'd really like to get in #624 
- [x] Less important but #621 would be useful

### CODE REVIEW

**Comment by lenemter** [comment]
In data/granite.appdata.xml.in on line 20 `Granite.STYLE_CLASS_BACKROUND` -> `Granite.STYLE_CLASS_BACKGROUND`

**Review by ryonakano — COMMENTED** [review]
Including [this change](https://github.com/elementary/granite/commit/694be994a66f91052106527f66a2bc064d6f6a02) in the appdata too and then we can release this.

**Comment by danirabbit** [comment]
@ryonakano done!

File: data/granite.appdata.xml.in [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <p>A companion library for GTK and GLib that provides complex widgets and convenience methods designed for use in apps built for elementary OS.</p>
   </description>
   <releases>
-    <release version="7.1.0" date="2022-09-27" urgency="medium">
+    <release version="7.1.0" date="2022-09-28" urgency="medium">
```
Comment (ryonakano) [inline data/granite.appdata.xml.in:13]: Let's update the date again and then it looks good for me.

**Review by ryonakano — APPROVED** [review]

**Pull request merged**